File: tests/openmp_examples/devices/teams.4.c
#pragma omp target map(to: B[0:1024*1024], C[0:1024*1024]) map(tofrom: sum)
#pragma omp teams num_teams(8) thread_limit(16) reduction(+:sum)
#pragma omp distribute parallel for reduction(+:sum) dist_schedule(static, 1024) schedule(static, 64)

File: tests/openmp_examples/devices/teams.4.f90
!$omp    target map(to: B, C) map(tofrom: sum)
!$omp    teams num_teams(8) thread_limit(16) reduction(+:sum)
!$omp    distribute parallel do reduction(+:sum) dist_schedule(static, 1024) schedule(static, 64)
!$omp    end teams
!$omp    end target
